demo/src/main/java/com/novoda/landingstrip/CustomLandingStrip.java

```diff
@@ -8,17 +8,24 @@
 
 public class CustomLandingStrip extends AppCompatActivity {
 
+    private LandingStrip landingStrip;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_custom_landing_strip);
         setTitle("Demo: " + getClass().getSimpleName());
 
-        LandingStrip landingStrip = (LandingStrip) findViewById(R.id.landing_strip);
+        landingStrip = (LandingStrip) findViewById(R.id.landing_strip);
         ViewPager viewPager = (ViewPager) findViewById(R.id.view_pager);
         viewPager.setAdapter(new DemoFragmentPagerAdapter(getSupportFragmentManager()));
 
         landingStrip.attach(viewPager, viewPager.getAdapter());
     }
 
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+        landingStrip.detach();
```

Comment:
Fixed in https://github.com/novoda/landing-strip/pull/26/commits/35c3263475ace6ee535c2fe1b602d8f5ea5b849b